taskcall/manifest.json

```
@@ -0,0 +1,71 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "dd54da03-0a8c-4796-aaa6-61eeb04e611b",
+  "app_id": "taskcall",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
```

Note: once you add the uninstallation instructions to the readme, you'll need to add the following line. 
```suggestion
    "configuration": "README.md#Setup",
    "uninstallation": "README.md#Uninstallation",
    "support": "README.md#Support",
```

---

Change corrected.